tmk_core/common/action_tapping.h

```diff
@@ -24,6 +24,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define TAPPING_TERM    200
 #endif
 
+#define RETRO_TAPPING // Tap anyway, even after TAPPING_TERM, as long as there was no interruption
```

Comment:
Let's leave this disabled by default until more folks have been able to try this out :)

Reply:
Agreed. Did that for easier testing for the reviewers. Thanks.

Reply:
Forgot to ask -- do I change that or do you do that?